galaxy_ng/app/access_control/fields.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth.models import Permission
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+from galaxy_ng.app.models import auth as auth_models
+
+
+class GroupPermissionField(serializers.Field):
```

Comment:
Class names makes false impression that field serializes single group or even a permission, while in fact it returns a list of groups with permissions and serializes not a single group but a list of groups.